app/controllers/ops_controller/settings/schedules.rb

```diff
@@ -26,6 +26,31 @@ def schedule_show
     end
   end
 
+  def schedule_run_now
+    schedules = find_records_with_rbac(MiqSchedule, checked_or_params)
+    schedules.each do |schedule|
+      MiqSchedule.queue_scheduled_work(schedule.id, nil, Time.now.utc.to_i, nil)
+      audit = {
+        :event        => "queue_scheduled_work",
+        :message      => "Schedule [#{schedule.name}] queued to run from the UI by user #{current_user.name}",
+        :target_id    => schedule.id,
+        :target_class => "MiqSchedule",
+        :userid       => current_user.userid
+      }
+      AuditEvent.success(audit)
+    end
+    unless flash_errors?
+      msg = if schedules.length == 1
+              _("The selected Schedule has been queued to run")
+            else
+              _("The selected Schedules have been queued to run")
+            end
```

Comment:
I think this would look better with a ternary operator, but as far as I know there's a method to have singular/plural forms of the same text based on a number. @mzazrivec can you help us here?

Reply:
Yes, it's much better to just use:
```ruby
msg = n_("The selected Schedule has been queued to run", "The selected Schedules have been queued to run", schedules.length)
```